backend/backend.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"path/filepath"
+	"strconv"
+
+	"gopkg.in/ini.v1"
+
+	"github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend/db"
```

Comment:
I don't recall needing to add github links like this, why can't you just directly link to the files in the repo?

Reply:
to use go mod